spec/system/sdg_management_spec.rb

```diff
@@ -0,0 +1,110 @@
+require "rails_helper"
+
+describe "SDGManagement" do
+  let(:user) { create(:user) }
+
+  before { Setting["feature.sdg"] = true }
+
+  context "Access" do
```

Comment:
I wonder whether we really need all these system tests :thinking:. Would we test the same behavior with some test of the `Abilities` models? System tests are much slower, after all :thinking:, so testing it with one rol who has access and one role who does not have access might be enough.